autonomy/data/Dockerfiles/agent/Dockerfile

```diff
@@ -13,20 +14,20 @@ RUN sudo apt-get update --fix-missing
 RUN sudo apt-get autoremove
 RUN sudo apt-get autoclean
 RUN sudo apt-get install python3.10 python3.10-dev git -y
-RUN sudo apt-get install net-tools subversion -y
-
 RUN cd /usr/bin && sudo rm python3 && sudo ln -s python3.10 python3 && sudo ln -s python3.10 python
+# used in Docker-compose to wait until Hardhat node is up
+RUN sudo apt-get install net-tools subversion -y
```

Comment:
Subversion is no longer needed right?

Reply:
yeah could probably remove it ...

Reply:
DO it, we dont want to vertical attacks :)